doc/controller_configuration/src/moveit_controller_manager_example.cpp

```diff
@@ -49,7 +49,7 @@ class ExampleControllerHandle : public moveit_controller_manager::MoveItControll
   {
   }
 
-  bool sendTrajectory(const moveit_msgs::RobotTrajectory& /*t*/) override
+  bool sendTrajectory(const moveit_msgs::RobotTrajectory& /*t*/, const ExecutionCompleteCallback& /*cb*/) override
   {
     // do whatever is needed to actually execute this trajectory
     return true;
```

Comment:
```suggestion
   // signal success
   cb(SUCCESS);
    return true;
```
or something in that vain?

Reply:
Okay!